### DISCUSSION TITLE
My IP has been blocked from raw.githubusercontent.com?

### DISCUSSION THREAD

**Discussion by nexxeln** [community/community#40527] [Projects and Issues]
### Select Topic Area

Bug

### Body

None of the gifs in any of the repositories are working. I think it's down, but in githubstatus.com it shows everything is operational.

Some of the repositories which use gifs in their readme but they aren't showing up. Pretty sure this is happening in every repo that uses a gif hosted in a asset folder on GitHub:

- [ajeetdsouza/zoxide](https://github.com/ajeetdsouza/zoxide)
- [koskimas/kysely](https://github.com/koskimas/kysely)

**Comment by jfullstackdev** [comment]
the GIFs are working on my end

![image](https://user-images.githubusercontent.com/47092464/205076845-b94ef5bd-eb12-4433-89e9-15aa7f765103.png)

![image](https://user-images.githubusercontent.com/47092464/205076795-bbdcb41e-3dc1-4241-94e7-611f6973f8fb.png)

maybe low computer memory, slow connection, GitHub extensions, cached data

try logging in in Incognito too after closing regular windows

**Reply by nexxeln** [reply]
Interesting. It's not loading for me in any browser, even on incognito.

**Reply by devarsh-mavani-19** [reply]
I tried clearing cache and cookies and also tried to open in incognito. But still does not work. I have lots of memory, and internet speed so that's not an issue. I havent added any github extensions. 

@nexxeln Did u found any solution?

**Reply by nexxeln** [reply]
> I tried clearing cache and cookies and also tried to open in incognito. But still does not work. I have lots of memory, and internet speed so that's not an issue. I havent added any github extensions.
> 
> @nexxeln Did u found any solution?

Yeah not working for me

**Comment by nexxeln** [comment]
Just tried on a different wifi network and it worked. Does that mean my ip address has been blacklisted? If yes, why?

**Reply by devarsh-mavani-19** [reply]
No you were not blacklisted. It was bug from their side. They have fixed it now. Try again from the old network. It started working again in mine.

**Reply by devarsh-mavani-19** [reply]
BTW are you based in india and do you use JIO network? It might have been an issue from internet provider end as well. I use JIO.

**Reply by nexxeln** [reply]
I do use JIO yes.

**Reply by nexxeln** [reply]
> No you were not blacklisted. It was bug from their side. They have fixed it now. Try again from the old network. It started working again in mine.

Still doesn't work on my end unfortunately.

**Reply by devarsh-mavani-19** [reply]
Yes so i guess JIO had issues. They might have blocked raw.githubusercontent.com for some reason or vice versa. Because it was not opening from other JIO simcards as well

**Reply by devarsh-mavani-19** [reply]
> > No you were not blacklisted. It was bug from their side. They have fixed it now. Try again from the old network. It started working again in mine.
> 
> Still doesn't work on my end unfortunately.

Might take a few hours. It will work. today morning it worked on 1 device but wasn't working on others. But now after 6-7 hours it is working everywhere. So wait for it I guess

**Reply by nexxeln** [reply]
Alright thanks. I'll wait and see

**Reply by devarsh-mavani-19** [reply]
It Stopped working in mine again :(

**Reply by N00b-User** [reply]
Any Workaround for JIO?

**Reply by jfullstackdev** [reply]
nope, as far as I know, GitHub does not block unless the location is really banned, 

and even in China, it can be accessed using VPN, so that's not GitHub side, it's your local setup either or your ISP or anything that can block GitHub, so please do tell those things so we can pin down what's causing it

**Reply by nexxeln** [reply]
I can confirm its JIO. I'm currently using cloudflare's [1.1.1.1](https://1.1.1.1) to work around this. Hopefully that's helpful @N00b-User

**Reply by aminoxix** [reply]
Yeah, it's for JIO only, it's not working for me as well : (

**Reply by bhupinder-bisht-sendinblue** [reply]
Yeah, it's for JIO only, it's not working for me as well

**Reply by Dhravya** [reply]
Jio wifi - it just stopped working for me too. 

It does work with cloudflare warp (1.1.1.1)

**Reply by dinesh-hills** [reply]
Same issue with my JIO. Where should I change the DNS, on control panel or JIO router configuration?

update: I have changed the ipv4 and ipv6, flushed my dns. Now it works fine.

**Reply by abeegit** [reply]
@dinesh-hills Do you mind telling us how you did it? I'm on ACT, not Jio btw

**Reply by dinesh-hills** [reply]
@abeegit Yeah update both your ipv4 and ipv6 DNS to cloudflare dns resolver. [https://developers.cloudflare.com/1.1.1.1/setup/windows/](url)

**Reply by dinesh-hills** [reply]
@Dhravya Actually it is working through **ipv6** so add ipv6 dns resolver in your windows / linux settings too

**Comment by dinesh-hills** [comment]
![image](https://user-images.githubusercontent.com/118039836/210249039-e14027e2-a293-499a-ac89-6d0c581aa0e7.png)
